### ISSUE TITLE
Spreadsheets for submissions before 15th June 2020 are missing

### ISSUE THREAD

**Issue by aaclan-ebi** [ebi-ait/hca-ebi-wrangler-central#185]

**Comment by aaclan-ebi** [comment]
It looks like that the spreadsheets weren't restored when we rebuild our prod cluster for Helm v3 upgrade last 15th Jun 2020. The volumes associated from the previous prod instances are already cleaned up as well. I'm afraid there's no way to easily recover the old spreadsheets.

The Updates spreadsheets feature was released to prod  last [17th July 2019] (https://docs.google.com/document/d/1xbOUnp4ZvVn0D-zGWtWxSyonVBK3mHueNHuJto-Iap8/edit#heading=h.134x72v7p2ug)

We're missing the spreadsheets stored from the first release date, 17th July 2019,  to  15th June 2020 when prod cluster was rebuilt.

**Comment by aaclan-ebi** [comment]
I checked the HCA DCP account, and i believe we can still easily recover the 2019 spreadsheets.

We're now just missing the 1st Jan to 15th June 2020 spreadsheets.

@clairerye may I know your thoughts on the severity of recovering this? We can manually upload the missing spreadsheets from the wrangler google drive. It'll be a little tricky and tedious, I may need to provide a way to upload the spreadsheets via UI/ or create a script to do this.  But I'm not sure if it's worth it. We might be able to support the spreadsheet generation from project content by the time they're needed. If that feature is not ready, we can just find them from google drive and recover them when there's a real need. I'll just work on the recovering the 2019 spreadsheets for now. Please let me know if you have any concerns on this.

**Comment by aaclan-ebi** [comment]
I checked the stored spreadsheets in the ingest-broker deployment and it looks like that it only contains spreadsheets from 2019-11-25 . It seems that all of those are the integration test spreadsheets. I think i'd have to further check the old volumes to recover 2019 spreadsheets.

**Comment by aaclan-ebi** [comment]
After further investigation, it looks like we can recover all spreadsheets from 18 July 2019. 

The created timestamp of the files were in the very recent available volume in the EBI AWS account. The volume used before the 15th June 2020  rebuild. I got confused by the created timestamps as it's showing very recent timestamps. Looks like the created timestamps of the files are being overwritten when they are being extracted. It was originally restored when we first deployed broker to EBI AWS account. 

TLDR:  I verified the submission uuids and there are spreadsheets for submission done for 18 July 2019. Still need some work to actually restore them in the current deployment.

**Comment by aaclan-ebi** [comment]
I've restored the spreadsheets (found from previously used volumes) of old submissions to the current broker deployment.

Interestingly, i still found some missing spreadsheets. Spreadsheets created from 28 Feb 2020 to 15 Jun 2020 are missing.

![Screenshot 2020-12-07 at 10.13.39.png](https://images.zenhubusercontent.com/5b042c754b5806bc2bd426e2/81df9c65-0f1a-4460-8ec6-5f6af3fce11b)

I'll just take note of these for now. We'd have to check the wrangler google drive should we need the spreadsheets. I will move on to other tasks.
